spec/api/api.yaml

```diff
@@ -126,15 +131,22 @@ paths:
         - API
       summary: Reset the test server
       description: |- 
-        Reset the test server by deleting all of the databases and re-creating new databases with prebuilt datasets if specified. 
+        Reset the test server by deleting all of the databases and re-creating new databases as specified. 
       operationId: reset
       requestBody:
         description: |- 
-          The request object describes how the databases will be setup after being reset. The properties include:
+          The request object describes how the databases will be setup after being reset. The request object
```

Comment:
Shouldn't this have a ref to an explicit schema?

Reply:
It has a reference to an explicit schema. I could move this description there.

Reply:
Done.